src/main/index.js

```diff
@@ -68,6 +68,8 @@ function init () {
   app.on('ready', function () {
     isReady = true
 
+    // require('electron').BrowserWindow.addDevToolsExtension('/Users/feross/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/0.15.3_0/')
```

Comment:
Yeah, sorry I'll remove that. It adds React Dev Tools to the Electron inspector.